llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -10,115 +10,129 @@
 // RISC-V subtarget features and instruction predicates.
 //===----------------------------------------------------------------------===//
 
+class RISCVExtension<string n, int major, int minor, string f, string v,
+                     string d, list<RISCVExtension> i = []>
```

Comment:
What if an extension needs to depend from a SubtargetFeature that is not a RISCV extension?